hmda-analytics/src/main/scala/hmda/analytics/query/TransmittalSheetComponent.scala

```diff
@@ -87,9 +87,6 @@ trait TransmittalSheetComponent {
     def deleteByLei(lei: String): Future[Int] =
       db.run(table.filter(_.lei.toUpperCase === lei.toUpperCase).delete)
 
-    def deleteByLeiAndQuarter(lei: String): Future[Int] =
```

Comment:
This function is redundant right now so I deleted it - not used anywhere that deleteByLei isn't